src/prngs/fortuna.c

```diff
@@ -8,6 +8,12 @@
  */
 #include "tomcrypt.h"
 
+#if defined(_WIN32)
+  #include <windows.h>
+#else
+  #include <sys/time.h>
```

Comment:
`sys/time.h` is not available on embedded micros, so there should be some `#if defined` to see if the user has defined their own time functions.